python/cugraph/cugraph/structure/graph_classes.py

```diff
@@ -134,6 +135,15 @@ def from_cudf_edgelist(
             Indicate whether or not to renumber the source and destination
             vertex IDs.
 
+        store_transposed : bool, optional (default=False)
+            If True, stores the transpose of the adjacency matrix.  Required
+            for certain algorithms.
+
+        legacy_renum_only : bool, optional (default=False)
+            If True, skips the C++ renumbering step.  Must be true for
+            pylibcugraph algorithms.  Must be false for algorithms
+            not yet converted to the pylibcugraph C API.
```

Comment:
We should revisit this in a separate PR since I'm thinking most users won't understand it, but for now we'll want this PR merged to unblock others.

Reply:
I filed [this issue](https://github.com/rapidsai/cugraph/issues/2437) so we don't forget.